inc/template-tags.php

```diff
@@ -7,6 +7,31 @@
  * @package _s
  */
 
+if ( ! function_exists( '_s_comment_nav' ) ) :
+/**
+ * Prints comment navigation.
+ */
+function _s_comment_nav() {
+	 // Are there comments to navigate through?
+	if ( get_comment_pages_count() > 1 && get_option( 'page_comments' ) ) : ?>
+		<nav class="navigation comment-navigation" role="navigation">
+			<h2 class="screen-reader-text"><?php esc_html_e( 'Comment navigation', '_s' ); ?></h2>
+			<div class="nav-links">
+				<?php
+					if ( $prev_link = get_previous_comments_link( __( 'Older Comments', '_s' ) ) ) :
+						printf( '<div class="nav-previous">%s</div>', esc_html( $prev_link ) );
```

Comment:
You are absolutely right. Don't know how I missed it. Will update the PR. Well, just in case...